src/view/base-template.twig

```diff
@@ -11,7 +11,6 @@
 <meta name="generator" content="Skosmos {{ request.version }}">
 <meta name="title" content="{{ block('title') }}">
 <meta property="og:title" content="{{ block('title') }}">
-<meta name="twitter:title" content="{{ block('title') }}">
 <meta name="twitter:card" content="summary">
```

Comment:
https://developer.x.com/en/docs/twitter-for-websites/cards/overview/markup#overview-of-all-twitter-card-tags

Now that we're getting rid of twitter:title in favour of og:title, we could also change twitter:card to og:type. I reckon the domain of og:type fits skosmos pages better than the domain of twitter:card.